geomstats/geometry/riemannian_metric.py

```diff
@@ -556,3 +598,25 @@ def sectional_curvature(self, tangent_vec_a, tangent_vec_b, base_point=None):
         condition = gs.isclose(normalization_factor, 0.0)
         normalization_factor = gs.where(condition, EPSILON, normalization_factor)
         return gs.where(~condition, sectional / normalization_factor, 0.0)
```

Comment:
I think we may simplify this to avoid the need to make a division for a value close to zero. Can be left for the future.

Reply:
sounds good!